fendermint/vm/genesis/src/lib.rs

```diff
@@ -16,6 +16,26 @@ use fendermint_vm_encoding::IsHumanReadable;
 #[cfg(feature = "arb")]
 mod arb;
 
+/// The genesis data structure we serialize to JSON and start the chain with.
+#[serde_as]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq)]
+pub struct Genesis {
+    /// The name of the blockchain.
+    ///
+    /// It will be used to derive a chain ID as well as being
+    /// the network name in the `InitActor`.
+    pub chain_name: String,
```

Comment:
This `String` is actually expected to be the stringinfied representation of a `SubnetID` right?